lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
         }
       // }
 
-      .info {
-        background: $content-wrapper-bg;
-        height: 220px + 130px;
-        left: 0px;
-        position: absolute;
-        top: 0px;
-        @include transition(all 0.15s linear 0s);
-        width: 100%;
-        overflow: hidden;
-
+      .course-image {
         .cover-image {
-          height: 200px;
+          height: 160px;
```

Comment:
@talbs,
I've fixed the height so that the image container has the same height, no matter the height of the image. Setting the `max-height` as the alternative would take care of all the cases in which the image height is greater than 160px, while images with height lesser than 160px would cause the container to shrink to the height of the image it's containing.

Since we want to preserve the height of the image and make the course cards that are all of the same height, I've decided to take this approach, but I'm open to any suggestions.